### PR TITLE
[ Storybook ] Install Storybook for design system - 712 & 713 [ Prod ]

### PR DESCRIPTION
**Pull request by bolade-oye**
# Description

Install Storybook, Vue3, and Vite (for the design system) on styles. It's a large PR and contains a lot of default files from storybook. I only made updates to the following:
 - `.gitignore`: Added recommended ignore options
 -  `README.md`: Updated it to include the command to run storybook
 - `package.json` files: Did a merge between already existing package.json file and new dependencies 
 
 
## Screenshot
 image

Fixes vc4a-dev/plan#712 & vc4a-dev/plan#713

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```
Comment (matt-vc4a) [inline index.html:1]: Is this file needed?

File: src/App.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<script setup>
```
Comment (matt-vc4a) [inline src/App.vue:1]: needed?

or is this the base you'll work from?

File: src/components/HelloWorld.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<script setup>
```
Comment (matt-vc4a) [inline src/components/HelloWorld.vue:1]: also needed?

File: src/stories/Button.vue [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    },
+  },
+
+  emits: ['click'],
+
+  setup(props, { emit }) {
+    props = reactive(props);
+    return {
+      classes: computed(() => ({
+        'storybook-button': true,
```
Comment (matt-vc4a) [inline src/stories/Button.vue:38]: why the `storybook-` prefix?

File: src/stories/Configure.mdx [hunk 5]
```diff
@@ -0,0 +1,364 @@
+import { Meta } from "@storybook/blocks";
```
Comment (matt-vc4a) [inline src/stories/Configure.mdx:1]: what's this? a tutorial page?

File: src/stories/Header.stories.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import MyHeader from './Header.vue';
```
Comment (matt-vc4a) [inline src/stories/Header.stories.js:1]: another tutorial/example?

File: src/stories/assets/accessibility.png
Comment (matt-vc4a) [inline src/stories/assets/accessibility.png:1]: Can these example files be removed? If we wait too long they'll stay there forever

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,79 @@
+:root {
```
Comment (matt-vc4a) [inline src/style.css:1]: if you remove the example files, don't forget their css too

**Review by matt-vc4a — CHANGES_REQUESTED** [review]
mainly questions about what to do with the demo files that come pre-made

File: index.html (hunk 1 above)
Comment (bolade-oye) [reply index.html:1]: Yes, This is needed for Vue to run

File: src/App.vue (hunk 2 above)
Comment (bolade-oye) [reply src/App.vue:1]: This is also needed for Vue to run

File: src/components/HelloWorld.vue (hunk 3 above)
Comment (bolade-oye) [reply src/components/HelloWorld.vue]: Removed. https://github.com/vc4a-dev/vc4a-styles/pull/194/commits/4bd81316722f32a9a21b531f0e44ffb697e4100e

File: src/stories/Button.vue (hunk 4 above)
Comment (bolade-oye) [reply src/stories/Button.vue:40]: It's a tutorial file https://github.com/vc4a-dev/vc4a-styles/pull/194/commits/9966dc63d530bd49f2a39fc1cc24d499e397e145

File: src/stories/Configure.mdx (hunk 5 above)
Comment (bolade-oye) [reply src/stories/Configure.mdx:3]: Yes, I've added a comment to the first line fo each tutorial file https://github.com/vc4a-dev/vc4a-styles/pull/194/commits/9966dc63d530bd49f2a39fc1cc24d499e397e145

File: src/stories/Header.stories.js (hunk 6 above)
Comment (bolade-oye) [reply src/stories/Header.stories.js:3]: Yes, Added a comment https://github.com/vc4a-dev/vc4a-styles/pull/194/commits/9966dc63d530bd49f2a39fc1cc24d499e397e145

File: src/stories/assets/accessibility.png
Comment (bolade-oye) [reply src/stories/assets/accessibility.png:1]: I'll remove them when taking out the tutorial files

**Review by matt-vc4a — APPROVED** [review]

**Pull request merged**